torch/_export/passes/lift_constants_pass.py

```diff
@@ -262,6 +262,10 @@ def lift_constants_pass(
                 node.replace_all_uses_with(const_placeholder_node)
                 gm.graph.erase_node(node)
 
+                for spec in graph_signature.output_specs:
+                    if spec.arg.name == node.target:
+                        spec.arg.name = const_placeholder_node.name
```

Comment:
maybe we can put a mapping of original node -> output spec ahead of time so that we don't have to loop through all the output specs every time we lift a constant?